src/services/query.ts

```diff
@@ -109,8 +107,12 @@ export class QueryTermsService {
   }
 }
 
-const alphabeticallyByPrefLabel = (a: Term, b: Term) => {
+const alphabeticallyByLabel = (a: Term, b: Term) => {
   const prefLabelA = a.prefLabels[0]?.value ?? '';
+  const altLabelA = a.altLabels[0]?.value ?? '';
+  const sortLabelA = prefLabelA.concat(altLabelA);
```

Comment:
I think in JavaScript it's a bit more common (and slightly faster) to use the string concat operator rather than  `concat()`, e.g. `const sortLabelA = prefLabelA + altLabelA;`

Reply:
I was on this track before try to operate it on the conditional statement directly but that didn't work. This is indeed a clearer approach.